src/core/or/circuitmux.h

```diff
@@ -80,6 +92,34 @@ struct circuitmux_policy_circ_data_s {
   uint32_t magic;
 };
 
+/*
+ * This struct holds whatever we want to store per attached circuit on a
+ * circuitmux_t; right now, just the count of queued cells and the direction.
+ */
+
+struct circuit_muxinfo_s {
```

Comment:
...but moving the struct definitions seems like a step backwards.  Generally we try to make structure definitions more hidden, and not expose them more.

(Then again, since I am reading these commits in order, I might wind up seeing that you've done this for a very good reason later on.  I'll wait and see...)